features/openhab-core/src/main/feature/feature.xml

```diff
@@ -63,6 +63,19 @@
         <bundle>mvn:org.openhab.core/org.openhab.core.compat1x/${project.version}</bundle>
     </feature>
 
+  <feature name="openhab-httpclient" version="${project.version}">
```

Comment:
It looks formatted differently 🙂

And do I understand correct that we upgrade karaf downgrade jetty

Reply:
Indeed - fixed now.

The upgrade of Karaf brings Jetty 9.3.21 instead of 9.3.15 from before - I thus reverted our "forceful Jetty upgrade to 9.3.22", which indeed results in a slight downgrade of the Jetty server.

But this feature here is actually about the Jetty client - I noticed that we so far included 9.3.15 as a client, so this change will actually upgrade it to 9.3.21 so that server + client version are in sync.